lib/elixir/test/elixir/kernel/impl_test.exs

```diff
@@ -24,6 +24,41 @@ defmodule Kernel.ImplTest do
     @macrocallback bar :: any
   end
 
+  defmodule OverridableBehaviour do
+    @callback foo :: any
+    @callback bar :: any
+
+    defmacro __using__(_) do
```

Comment:
You don't need to define those here. You can define them in the test itself before your own clauses. The Elixir code doesn't know that something is happening before or after use. :)